corehq/apps/domain/models.py

```diff
@@ -1227,3 +1229,46 @@ def as_dict(self):
             'deactivate_url': self.deactivate_url(),
             'activate_url': self.activate_url(),
         }
+
+
+class DomainAuditRecordEntry(models.Model):
+    id = models.UUIDField(unique=True, default=uuid.uuid4, primary_key=True)
```

Comment:
Is there a reason we need to set the primary key ```id``` to a UUID field instead of using the built-in django id primary field?

Reply:
I thought that we should use ```UUID``` in the primary keys.